regression/contracts/invar_struct_member/main.c

```diff
@@ -0,0 +1,19 @@
+struct test
+{
+  int x;
+};
+
+void main()
+{
+  struct test t;
+  t.x = 0;
+
+  unsigned n;
+  for(unsigned i = 0; i < n; ++i)
+    __CPROVER_loop_invariant(i <= n)
+    {
+      t.x += 2;
+    }
+
+  assert(t.x == 0 || t.x == 2);
```

Comment:
Why this assertion?

Reply:
This assertion would pass if `t.x` is not havoced (so stays at value `0`), but should fail if it's havoced.

I will add a comment explaining this in the `.desc` file.